docs/content/doc/usage/https-support.md

```diff
@@ -76,5 +76,6 @@ After that, enable HTTPS by following one of these guides:
 * [apache2/httpd](https://httpd.apache.org/docs/2.4/ssl/ssl_howto.html)
 * [caddy](https://caddyserver.com/docs/tls)
 
-Note: Your connection between your reverse proxy and Gitea might be unencrypted. To encrypt it too, follow the [built-in server guide](#using-built-in-server) and change
-the proxy url to `https://[URL]`.
+Note: Enabling HTTPS by this method is referred as [TLS Termination Proxy](https://en.wikipedia.org/wiki/TLS_termination_proxy). The reverse proxy handles TLS connections, and decrypted them for passing to Gitea. This is fine as long as both the proxy and Gitea instances are either on the same machine, or on different machines at private network (with the proxy is exposed to outside network. If your Gitea instance is at public network instead, you might also consider to [enable HTTPS using built-in server](#using-the-built-in-server) to encrypt connections from the proxy to Gitea.
```

Comment:
```suggestion
Note: Enabling HTTPS only at the proxy level is known as a [TLS Termination Proxy](https://en.wikipedia.org/wiki/TLS_termination_proxy). The proxy server accepts incoming TLS connections, decrypts the contents, and passes the now unencrypted contents to the Gitea application. This is normally fine as long as both the proxy and Gitea instances are either on the same machine, or on different machines within the same private network. If your Gitea instance is separated from your Proxy over a public network, or you want full end-to-end encryption, you can also enable HTTPS support directly in Gitea [using the built-in server](#using-the-built-in-server) and forward the connections over HTTPS instead.
```
Some small adjustments for English/clarity

Reply:
OK